package.json

```diff
@@ -2,12 +2,15 @@
   "name": "great-novels-ui",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "server.js",
   "scripts": {
-    "dev:server": "./node_modules/.bin/env-cmd ./node_modules/.bin/nodemon index.js",
+    "dev": "./node_modules/.bin/concurrently \"npm run dev:client\" \"npm run dev:server\"",
+    "dev:client": "./node_modules/.bin/webpack --progress --inline --config webpack.dev.js",
     "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./configs/sequelize.js",
+    "dev:server": "./node_modules/.bin/env-cmd ./node_modules/.bin/babel-watch server.js",
     "lint": "./node_modules/.bin/eslint --format codeframe .",
```

Comment:
You've got some linting errors

![image](https://user-images.githubusercontent.com/5775716/84533763-cc4f5d00-acb6-11ea-9e0e-414efe3ce007.png)